avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -542,9 +546,12 @@ def updatePositionC(cfg, particles, dem, force, DT):
   cdef double TotkinEne = particles['kineticEne']
   cdef double TotpotEne = particles['potentialEne']
   cdef double peakKinEne = particles['peakKinEne']
+  cdef double peakForceSPH = particles['peakForceSPH']
+  cdef double TotkForceSPH = particles['forceSPHIni']
```

Comment:
I suggest to start using the correct naming style: totKForceSPH...

Reply:
done